http/server_test.go

```diff
@@ -1,13 +1,13 @@
 package http_test
 
 import (
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-	"github.com/kkrull/gohttp/http"
-	"net"
 	"bytes"
+	"github.com/kkrull/gohttp/http"
 	"github.com/kkrull/gohttp/mock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
 	"io/ioutil"
+	"net"
```

Comment:
As long as we're cleaning up imports, perhaps consider ordering them with blank lines as described [here](https://github.com/golang/go/wiki/CodeReviewComments#imports)